sdk/js/src/Client.ts

```diff
@@ -267,7 +267,7 @@ export class DVCClient implements Client {
 
     track(event: ClientEvent): void {
         if (this._closing) {
-            this.logger.error('Client is closing, cannot track new events.')
+            this.logger.warn('Client is closing, cannot track new events.')
```

Comment:
this one actually is an error, it's a user problem

Reply:
bump

Reply:
just fixed! 😂